src/components/IconBar/IconBar.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import SettingsIcon from '@material-ui/icons/Settings';
+import TownhallIcon from '@material-ui/icons/AccountBalance';
+
+
+import { DeviceContext } from 'contexts/Device';
+
+import Component from './IconBar';
+
+export default { title: 'Icon Bar' };
+
+export function CenterIconBar() {
+    return (
+        <Component justify={'center'}>
```

Comment:
This should be giving you a linting error (I think)

`<Component justify='center'`

Don't forget to change the other ones too and make sure you've run `yarn install` then `yarn lint` and get rid of any linting errors that have to do with your code.

Reply:
Resolved